buddhi/lib/amp/toolkit/buddhi/profiles/simple.rb

```diff
@@ -0,0 +1,30 @@
+module AMP
+  module Toolkit
+    module Buddhi
+      module Profiles
+        class Simple
+          def call(client, endpoint:, **_options)
+            service = ThreeScale::Helper.create_service(client)
+            plan = ThreeScale::Helper.create_application_plan(client, service)
+            ThreeScale::Helper.create_application_plan_limit(client, service, plan)
```

Comment:
What's the motivation of creating an application plan limit?

Reply:
Plans limits are part of the traffic model. The more limits, the more jobs 3scale backend has to do, the more load internal redis receives. 

Currently, the `simple` model creates just one limit. But we could decide to remove it. It is just "simple" traffic model definition.

Reply:
:+1: